### DISCUSSION TITLE
Failure message at bottom of form?

### DISCUSSION THREAD

**Discussion by TUSAB-web** [solspace/craft-freeform#423] [Questions & Answers] [answered]
By default, when somebody hits the “Submit” button on the form, if there’s an error (like a required field not filled out), that error shows at the TOP of the form. Is there a way to have that message display at the bottom of the form?

**Comment by kjmartens** [comment, marked as answer]
Hi @TUSAB-web,

We have a little example/guide for this here: 🙂 
https://docs.solspace.com/craft/freeform/v3/templating/ajax-forms.html#display-success-banner-at-bottom

**Reply by TUSAB-web** [reply]
Thanks - It looks like I can just modify that to display a failure message instead!
